stdlib/multiprocessing/managers.pyi

```diff
@@ -76,9 +76,21 @@ class Server:
     def accept_connection(self, c: Connection, name: str) -> None: ...
 
 class BaseManager:
-    def __init__(
-        self, address: Any | None = ..., authkey: bytes | None = ..., serializer: str = ..., ctx: BaseContext | None = ...
-    ) -> None: ...
+    if sys.version_info >= (3, 11):
+        def __init__(
+            self, address: Any | None = ..., authkey: bytes | None = ..., serializer: str = ..., ctx: BaseContext | None = ...
+        ) -> None: ...
+    else:
+        def __init__(
+            self,
+            address: Any | None = ...,
+            authkey: bytes | None = ...,
+            serializer: str = ...,
+            ctx: BaseContext | None = ...,
+            *,
+            shutdown_timeout: float = ...,
+        ) -> None: ...
```

Comment:
Shouldn't these be the other way around? 🤔

Reply:
Indeed, fixed. I think that's a sign I've done enough of these for today.

Reply:
Are you using stubtest to find these, out of interest? :)

Reply:
No, just going over `git log` in CPython